scripts/bootstrap-ansible.sh

```diff
@@ -64,8 +64,19 @@ pushd ${OA_DIR}
     exit 99
   fi
 
-  # Now use GROUP_VARS of OSA and RPC
-  sed -i "s|GROUP_VARS_PATH=.*|GROUP_VARS_PATH=\"\${GROUP_VARS_PATH:-${OA_DIR}/playbooks/inventory/group_vars/:${BASE_DIR}/group_vars/:/etc/openstack_deploy/group_vars/}\"|" /usr/local/bin/openstack-ansible.rc
-  sed -i "s|HOST_VARS_PATH=.*|HOST_VARS_PATH=\"\${HOST_VARS_PATH:-${OA_DIR}/playbooks/inventory/host_vars/:${BASE_DIR}/host_vars/:/etc/openstack_deploy/host_vars/}\"|" /usr/local/bin/openstack-ansible.rc
+  # RPC-O has roles in its own git tree, so we need to add it to the
+  # path for Ansible to search.
+  sed -i "s|/etc/ansible/roles:|/etc/ansible/roles:${RPCD_DIR}/playbooks/roles:|" /usr/local/bin/openstack-ansible.rc
+
+  # NOTE(cloudnull): Sync {group,host}_vars from rpc-o into the proper OSA
+  #                  directory which supports the default override interface.
+  if [[ -d "${BASE_DIR}/group_vars" ]];then
+    rsync -av ${BASE_DIR}/group_vars /etc/openstack_deploy/
```

Comment:
I would suggest including the use of ``--delete`` to remove any files removed in-tree too, then add a release note to note to deployers that any use of ``/etc/openstack_deploy/group_vars`` will be overwritten and that overrides should only be done in the user_* files.

Reply:
done.